bin/oref0-setup.sh

```diff
@@ -70,6 +75,10 @@ case $i in
     ;;
 esac
 done
+if [[ -z $TTY && radio_locale != "US" ]]; then
+    echo "World wide radio locale is only supported with TI stick currently"
```

Comment:
That's correct. The Carelink sticks come in USA and WW variants - and need to match the pump. You don't need to handle it specifically.